_build/data/transport.core.system_settings.php

```diff
@@ -1969,4 +1969,22 @@
     'area' => 'site',
     'editedon' => null,
 ), '', true, true);
+$settings['uberbar'] =  $xpdo->newObject('modSystemSetting');
+$settings['uberbar_mode']-fromArrqy(array (
+     'key' => 'uberbar_mode'
+    ,'value' => 'simple'
+    ,'xtype' => 'textfield'
+    ,'namespace' => 'core'
+    ,'area' => 'uberbar'
+    ,'editedon' => null
+), '', true, true);
+$settings['uberbar'] =  $xpdo->newObject('modSystemSetting');
+$settings['uberbar_maxresults']-fromArrqy(array (
```

Comment:
`$settings['uberbar_maxresults']-fromArrqy(` should be `$settings['uberbar_maxresults']->fromArray(`